Global.json

```diff
@@ -1,5 +1,6 @@
 {
```

Comment:
The version of the sdk is fixed in the CI. So, this file is only for local environment. Instead of updating it regularly I think we can just remove it. As long as you have a recent sdk it should work.

BTW can you update the SDK in `azure-pipeline.json` to 3.1.200?